### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception com.intellij.ide.ui.UITheme$1@271ca770 cannot patch icon path java.lang.StringIndexOutOfBoundsException: String index out of range: 0

### ISSUE THREAD

**Issue by yangbing0728** [microsoft/azure-tools-for-java#6674]
IntelliJ build version: 2021.3.2 IU-213.6777.52
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.13
Plugin version: 3.65.0-2021.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1440x812,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.util.ui.UIUtil$1@2fbe7ec4,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=com.intellij.ide.ui.UITheme$1@271ca770 cannot patch icon path, throwable=java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.base/java.lang.StringLatin1.charAt(StringLatin1.java:47)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.loadIcon(IntelliJAzureIcons.java:109)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.lambda$doGetIcon$2(IntelliJAzureIcons.java:91)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.doGetIcon(IntelliJAzureIcons.java:91)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.getIcon(IntelliJAzureIcons.java:82)
	at com.microsoft.azure.toolkit.intellij.common.action.IntellijAzureActionManager$AnActionWrapper.applyView(IntellijAzureActionManager.java:139)
	at com.micro

**Comment by Flanker32** [comment]
@yangbing0728 Thanks for your report, the issue should be resolved in toolkit v3.67.0, could you please upgrade to the latest version and try again?

**Comment by wangmingliang-ms** [comment]
duplicate of #6758
